Sources/MessagingInApp/Gist/Gist.swift

```diff
@@ -38,41 +37,42 @@ public class Gist: GistDelegate {
         clearUserToken()
         messageQueueManager = MessageQueueManager()
         messageManagers = []
-        RouteManager.clearCurrentRoute()
+        // RouteManager.clearCurrentRoute()
     }
 
     // MARK: User
 
     public func setUserToken(_ userToken: String) {
-        UserManager().setUserToken(userToken: userToken)
+        // UserManager().setUserToken(userToken: userToken)
     }
 
     public func clearUserToken() {
         cancelModalMessage(ifDoesNotMatchRoute: "") // provide a new route to trigger a modal cancel.
         messageQueueManager.clearLocalStore()
-        UserManager().clearUserToken()
+        // UserManager().clearUserToken()
         messageQueueManager.clearUserMessagesFromLocalStore()
```

Comment:
does `messageQueueManager` also get removed in later PRs, cause I don't see it commented here?

Reply:
`messageQueueManager` doesn't get removed, but the call does. You can take a look at PR #802 to see the final implementation.